packages/ds-healthcare-gov/src/components/Header/defaultMenuLinks.ts

```diff
@@ -1,41 +1,32 @@
 import { VARIATION_NAMES } from './Header';
 import localeLink from './localeLink';
 import loginLink from './loginLink';
-import { translate } from '../i18n';
+import { TFunction } from 'i18next';
+
+export const LINK_IDENTIFIERS = {
+  LOGIN: 'login',
+  LOGOUT: 'logout',
+};
 
 /**
  * Default menu links for each header variation.
  * Apps can import this method into their app if they need to
  * extend the existing default list of menu links.
- * @param {Function} translate - i18n translator
- * @param {String} locale
- * @param {Boolean} deConsumer
- * @param {String} subpath
- * @param {String} primaryDomain
- * @param {String} switchLocaleLink
- * @param {boolean} hideLoginLink
- * @param {boolean} hideLanguageSwitch
- * @param {boolean} customLinksPassedIn
- * @returns {Object}
  */
 export function defaultMenuLinks(
+  t: TFunction,
   locale = 'en',
-  deConsumer,
-  subpath,
+  deConsumer?: boolean,
+  subpath?: string,
   primaryDomain = '',
```

Comment:
should primaryDomain have a `?` here? It seems odd that a required param is in the middle of a bunch of optional ones.

Maybe it's okay because it's defaulted, but seemed a little odd

Reply:
TypeScript won't allow a question mark if there's an intiailizer, but I could be explicitly defining the type as string